src/main/java/ca/xqz/tweetmouth/Ingest.java

```diff
@@ -0,0 +1,30 @@
+package ca.xqz.tweetmouth;
+
+import ca.xqz.tweetmouth.Tweet;
+import ca.xqz.tweetmouth.TweetParser;
+
+import java.io.IOException;
+
+import java.util.List;
+
+import org.elasticsearch.client.transport.TransportClient;
+
+class Ingest {
```

Comment:
Don't think we need the Ingest class. This was just the main function from TweetParser, which I was using as a test-main rather than a real main since the primary functionality is all in TweetLoader, which parses and then loads tweets to ES.